libraries/AP_Camera/examples/send_camera_information.py

```diff
@@ -0,0 +1,106 @@
+import threading
+import time
+from pymavlink import mavutil
+
+ONBOARD_CONTROLLER_SYSID = 245
+ONBOARD_CONTROLLER_COMPID = 0
+
+# Function to connect to MAVLink
+def connect_to_mavlink(ip, port):
+    connection = mavutil.mavlink_connection(f'udp:{ip}:{port}', source_system=ONBOARD_CONTROLLER_SYSID)
+    connection.wait_heartbeat()
+    print("Heartbeat received from system (system %u component %u)" % (connection.target_system, connection.target_component))
+    # Send a heartbeat back to register the device
+    connection.mav.heartbeat_send(
+        type=mavutil.mavlink.MAV_TYPE_ONBOARD_CONTROLLER,
+        autopilot=mavutil.mavlink.MAV_AUTOPILOT_INVALID,
+        base_mode=0,
+        custom_mode=0,
+        system_status=mavutil.mavlink.MAV_STATE_UNINIT,
+        mavlink_version=3
+    )
+    return connection
+
+ip = "127.0.0.1"
+port = 14570
+
+# Connect to MAVLink
+connection = connect_to_mavlink(ip, port)
```

Comment:
This will end up something like `self.connection = self.connect_to_mavlink(ip, port)`